pandas/core/sorting.py

```diff
@@ -223,7 +223,9 @@ def is_int64_overflow_possible(shape) -> bool:
     return the_prod >= lib.i8max
 
 
-def decons_group_index(comp_labels, shape):
+def _decons_group_index(
```

Comment:
Just curious on the rename here - why specifically make this one lead with an underscore?

Reply:
this is the only thing that stood out to me as being privatize-able